src/tiled/mapdocument.cpp

```diff
@@ -996,6 +996,39 @@ void MapDocument::moveObjectsToGroup(const QList<MapObject *> &objects,
     mUndoStack->endMacro();
 }
 
+// TODO add undo code
```

Comment:
If you want to look into adding this, note that there is already a `ChangeMapObjectsOrder` undo command that you can use for this purpose. You can see how it is used in the `RaiseLowerHelper`.